### DISCUSSION TITLE
How to get "flash" messages to render

### DISCUSSION THREAD

**Discussion by pboling** [janko/rodauth-rails#294] [Q&A] [answered]
I have the scaffoled / generated rodauth-rails views in my app, and I don't see any references to flash messages in them.

I tried enabling roda's `flash` plugin.

I don't see flash messages, though I assume they are being set.  It seems like I must have missed something obvious, but the only documentation I can find is to set [flash messages in the routing tree](https://fiachetti.gitlab.io/mastering-roda/#:~:text=Session%3A%3ACookie.-,flash,-One%20common%20need), which isn't what I'm after.

**Comment by janko** [comment, marked as answer]
rodauth-rails changes Rodauth to use Rails' flash messages instead of Roda's, so they're accessible in Rails views just like any other flash messages (`:notice` and `:alert` are used). Not sure if I should mention this behavior in the docs somewhere, the `rodauth:install` instructions mention that, though.

**Reply by pboling** [reply]
I was confused because this Rails app long ago ripped out the internal Rails' flash tooling, so my Roda app is the first thing to use it, and I wasn't sure how to get it.  This makes sense though.  I think it will work if I just re-add the standard Rails flash messages.
